### ISSUE TITLE
News stream

### ISSUE THREAD

**Issue by Gormartsen** [backdrop-ops/backdropcms.org#255]
We get to the point when every week few more modules or themes added.
New releases of contrib modules/themes or layouts came out.

It's hard to follow and see this activity.

We have next activities that nice to have more visible:
  - new module/theme/layout released
  - new release pushed

How about next:
- news page for contrib activities
- twitter posts for such activities
- gitter integration on backdrop-issusues page
- fb posts?
- email subscription ?

all, except gitter, can use tags:
 - project name
 - project type
 - activity type (new release, new project)

Benefits:
- more social activity on fb and tw
- easy to follow such activities using most popular social clients
- easy to share it and make more noise.

**Comment by jenlampton** [comment]
* FB & TWitter: We do have twitter posts about new modules, themes & layouts as they are released (and that auto-posts to FB as well).  

* Gitter: it will make more sense to have these updates go to the backdrop-contrib group, but we haven't really split our gitter channels yet since we don't yet have too many people for a single channel.

* News posts: I'm not sure we want posts on the website because it would be a lot of noise (and drown out all the real news), but we do have an embedded twitter feed on the news page. 

* Email: I *think* people can subscribe to backdrop-ops via github directly, but it might be a good idea to make that more easily available, though I don't know how.

**Comment by Gormartsen** [comment]
> FB & TWitter: We do have twitter posts about new modules, themes & layouts as they are released (and that auto-posts to FB as well).

Is it working for contrib? I don't see any tweets about new contrib releases or new contrib modules.

Basically idea is making this progress more visible.

**Comment by jenlampton** [comment]
Maybe it's not working, but we did set up a special feed for it: https://backdropcms.org/project/rss.xml

**Comment by jenlampton** [comment]
@thejimbirch can you check on the new-project tweet magic?

**Comment by thejimbirch** [comment]
Expect a week of Project twitter updates!

I added that feed to Buffer @jenlampton

**Comment by jenlampton** [comment]
awesomesauce!
